### PR TITLE
Add 0.14 support and update new syntax

### PR DESCRIPTION
**Pull request by snowsky**
## what
* Add Terraform 0.14 support
* Use new syntax in main.tf/versions.tf

## why
* This module needs to support Terraform 0.14

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -84,7 +85,8 @@ resource "aws_lb" "default" {
 }
 
 module "default_target_group_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source     = "cloudposse/null-label/aws"
```
Comment (aknysh) [inline main.tf:88]:
```suggestion
  source     = "cloudposse/label/null"
```

For `terraform-null-label` module, the provider is `null`

**Review by aknysh — CHANGES_REQUESTED** [review]
@snowsky  thanks for the PR!
Looks good, just one change, please see comments.
Also, please run the following commands in terminal:

```
make init
make github/init
make readme
```

It will update the repo with the latest GitHub actions and workflows we are adding to all modules.

Thans

**Comment by snowsky** [comment]
> @snowsky thanks for the PR!
> Looks good, just one change, please see comments.
> Also, please run the following commands in terminal:
> 
> ```
> make init
> make github/init
> make readme
> ```
> 
> It will update the repo with the latest GitHub actions and workflows we are adding to all modules.
> 
> Thanks
@aknysh updated, please see if this works, thanks. :) It is great to know more how the repos are maintained.

**Review by aknysh — APPROVED** [review]
thanks @snowsky

**Pull request merged**